veba-bom.json

```diff
@@ -1,6 +1,6 @@
 {
     "veba": {
-        "version": "v0.7.5"
+        "version": "v0.8.0"
```

Comment:
Please revert back to `v0.7.5` as the VEBA version changes only occurs when we do a release

Reply:
Done